docs/docs/modifying-a-starter.md

```diff
@@ -0,0 +1,249 @@
+---
+title: Modifying a Starter
+---
+
+While a Gatsby [starter](/docs/starters/) is a working website out of the box, chances are you'll want to modify your starter at some point. Building on top of a starter reduces the amount of boilerplate code you need to write yourself. You're therefore able to write your own content, functionality, and styling faster.
+
+## Prerequisites
+
+What you need to know will depend on the starter you choose and the data or functionality you'd like to modify. Even if you choose _not_ to modify the starter's components, you may still want to update text, use data from an external source, and modify the style (CSS) of the site. To do this, you'll write some [Markdown](/docs/mdx/markdown-syntax/) and [JSON](https://www.digitalocean.com/community/tutorials/an-introduction-to-json).
+
+To modify the functionality of a starter, you'll want a basic understanding of [JSX](/docs/glossary#jsx) syntax for updating components and making new ones. You'll also want some knowledge of [GraphQL](/docs/graphql-concepts/) for querying your data. Start with these and add to your skills as you continue to add functionality to your starter.
+
+## Choosing a starter
+
+To choose a starter, first consider the functionality you need. Are you building an e-commerce site? A blog? Do you already know what data sources you'll want to use? Find a starter that fulfills your requirements by using the searchbar and filters in the [Gatsby Starter Library](/starters/).
+
+If you're not sure what to choose or want only the most essential functionality, try customizing either [`gatsby-starter-blog`](https://github.com/gatsbyjs/gatsby-starter-blog) (if you're primarily using this site as a blog) or [`gatsby-starter-default`](https://github.com/gatsbyjs/gatsby-starter-default). These official starters are maintained by Gatsby and are great options, particularly for your first Gatsby site.
+
+## Exploring the folder structure
+
+Taking the default Gatsby starter as an example, dedicate some time to looking through some of the most important files in the repository (repo). You'll find most of these in the root (or _main_) folder of your project. Doing so will give you a clearer picture of how the various parts of your Gatsby site work together.
+
+![Default starter file structure in VS Code](./images/default-starter-vs-code-focus.png)
```

Comment:
Thanks for your patience on this -- I think we should try one more take at the image here, especially since the previous paragraph talks about a "clearer picture". The sidebar is pretty small now with the whole VS Code interface; is it possible to crop it to the file list without the app buttons, header, or footer? I'd recommend keeping the file view area clear instead of blurring it, potentially with `gatsby-config.js` open for additional context. What do you think of something like this (I couldn't seem to increase the sidebar font size independently from the file area)?

![VS Code cropped](https://user-images.githubusercontent.com/1045233/67723765-31f60f80-f99a-11e9-83cd-1793a66d9ffa.png)

Reply:
I wound up zooming in and then making the editor font smaller. Thoughts on this one? Just sending the image since my connection is pretty spotty right now. Everything's taking longer than usual. 😅
<img width="1178" alt="default starter's files and contents of gatsby config" src="https://user-images.githubusercontent.com/7365296/67733021-dea4b100-f9db-11e9-87e9-b4e759ae00be.png">

Reply:
This is definitely an improvement. It's hard to fit it all in! I'd be good with this one, but I'll let @marcysutton have the final say.

Reply:
Thanks, @laurieontech, and congratulations!! I just pushed the project up with that screenshot since we're about to get back on the road. We should arrive in Curitiba in about 5 hours. We'll be staying in a hotel there, so fingers crossed for more reliable WiFi!

Reply:
I think this works! let's go for it!